examples/demo.js

```diff
@@ -208,10 +213,10 @@ window.onload = function () {
   }
 
   function downloadGif(image) {
-    download(image, 'index.gif', 'image/gif');// downloadjs library function
+    download(image, 'index.gif', 'image/gif'); // Downloadjs library function.
```

Comment:
Included purpose of downloadGif Function

Reply:
I didn't get you?